datarepo-client/build.gradle

```diff
@@ -33,19 +33,15 @@ dependencies {
 
     }
 
+    // TODO: the comment below seems wrong, since we are version locking these.
     // Swagger deps, versions controlled by dependencyManagement in root project
     implementation group: "io.swagger", name: "swagger-annotations", version: "${swagger}"
-    implementation group: "org.glassfish.jersey.core", name: "jersey-client"
-    implementation group: "org.glassfish.jersey.media", name: "jersey-media-json-jackson"
-    implementation group: "org.glassfish.jersey.media", name: "jersey-media-multipart"
-    implementation group: "com.fasterxml.jackson.datatype", name: "jackson-datatype-jsr310"
-    implementation group: "io.swagger.core.v3", name: "swagger-annotations"
+    implementation group: 'org.glassfish.jersey.core', name: 'jersey-client', version: "${jersey}"
+    implementation group: 'org.glassfish.jersey.media', name: 'jersey-media-json-jackson', version: "${jersey}"
+    implementation group: 'org.glassfish.jersey.media', name: 'jersey-media-multipart', version: "${jersey}"
+    implementation group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jsr310', version: "${jackson}"
+    implementation group: "io.swagger.core.v3", name: "swagger-annotations", version: "${swaggerAnnotations}"
```

Comment:
Both single and double quotes. Is it worth being consistent here?

Reply:
The rules are like shell, so in this case it doesn't matter. At one point I had normalized them all, but the style didn't stick.